cypress/page_object/task3/Footer.js

```diff
@@ -0,0 +1,16 @@
+class Footer {
+
+    // iconClass: '.em-facebook' , '.em-youtube' , '.em-instagram'
```

Comment:
i suppose this is for clarity, right? I'd recommend storing them in constants somewhere in another file
like this. it will increase usability and readability of the code
![image](https://github.com/user-attachments/assets/60f5b716-73c7-4090-8d22-fb87a7a54c67)